crates/bevy-inspector-egui/src/reflect_inspector/mod.rs

```diff
@@ -1223,7 +1219,7 @@ impl<'a, 'c> InspectorUi<'a, 'c> {
         &mut self,
         variant: &VariantInfo,
         ui: &mut egui::Ui,
-        enum_type_name: &'static str,
+        _enum_type_name: &'static str,
```

Comment:
I would remove this parameter altogether

Reply:
I agree. Removing.